pkgs/applications/networking/aether/default.nix

```diff
@@ -0,0 +1,89 @@
+{ autoPatchelfHook, makeDesktopItem, lib, stdenv, wrapGAppsHook
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cairo, cups, dbus, expat, fontconfig
+, freetype, gdk-pixbuf, glib, gtk3, libcxx, libdrm, libnotify, libpulseaudio, libuuid
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, libxshmfence
+, mesa, nspr, nss, pango, systemd, libappindicator-gtk3, libdbusmenu
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aether";
+  version = "2.0.0-dev15";
+  binaryName = "AetherP2P";
+  desktopName = "Aether";
+
+  src = fetchurl {
+    url = "https://static.getaether.net/Releases/Aether-2.0.0-dev.15/2011262249.19338c93/linux/Aether-2.0.0-dev.15%2B2011262249.19338c93.tar.gz";
+    sha256 = "1hi8w83zal3ciyzg2m62shkbyh6hj7gwsidg3dn88mhfy68himf7";
+    name = "aether-tarball.tar.gz";
+  };
+
+  dontWrapGApps = true;
+
+  nativeBuildInputs = [
+    alsa-lib
+    autoPatchelfHook
```

Comment:
that goes definitely into native.

Reply:
What do you mean by that? Is there something I should do?

Reply:
I think he just means `autoPatchelfHook` is in the right place. I agree.

Reply:
Yeah and almost everything else is buildInputs.